### PR TITLE
Add URL support and migrate to fec config

### PR DESCRIPTION
**Pull request by karelhala**
Screenshot from 2022-04-29 08-55-41

### CODE REVIEW

File: fec.config.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  /**
+   * Change to false after your app is registered in configuration files
+   */
+  interceptChromeConfig: false,
+  /**
+   * Add additional webpack plugins
+   */
+  plugins: [],
+  sassPrefix: '.api-docs, .apiDocs',
+  routes: {
```
Comment (Hyperkid123) [inline fec.config.js:15]: Can you disable the proxy config?

File: package.json [hunk 2]
```diff
@@ -74,15 +74,15 @@
...
   },
   "scripts": {
-    "build": "webpack --config config/prod.webpack.config.js",
+    "build": "fec build",
     "test": "jest --verbose",
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint config src",
     "lint:js:fix": "eslint config src --fix",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
     "prod": "NODE_ENV=production webpack serve --config config/dev.webpack.config.js",
```
Comment (Hyperkid123) [inline package.json:83]: There still some scripts referencing the old webpack config

File: src/routes/Detail.js [hunk 3]
```diff
@@ -111,6 +120,13 @@ const Detail = ({ loadApi, detail }) => {
                   deepLinking
                   docExpansion="list"
                   spec={detail.spec}
+                  requestInterceptor={async (req) => {
```
Comment (Hyperkid123) [inline src/routes/Detail.js:123]: Can you define the interceptor with the `useCallback` somewhere in the component? Better than recreating it on each render.

**Review by Hyperkid123 — COMMENTED** [review]
Looks good. Is there a config we can test this on?

**Comment by karelhala** [comment]
You can add
```YAML
api:
  url: https://api.openshift.com/api/kafkas_mgmt/v1/openapi
```

**Comment by Hyperkid123** [comment]
@karelhala the data is loaded fine, but the API endpoint is a bit strange. It uses the whole path:
```
https://api.openshift.com/api/kafkas_mgmt/v1/openapi
```

But I think it should be only

```
https://api.openshift.com/api/kafkas_mgmt/v1
```

Can we fixup the URL in the table?

screenshot-stage foo redhat com_1337-2022 04 28-09_38_01

Also the whole table looks strange we might have to bump PF.

File: fec.config.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+console.log(process.env, 'this is env!');
```
Comment (Hyperkid123) [inline fec.config.js:1]: I missed the console log!

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**